### DISCUSSION TITLE
Change the tree view location?

### DISCUSSION THREAD

**Discussion by BellZwei** [lllyasviel/stable-diffusion-webui-forge#1426] [Q&A] [answered]
I was wondering if there is an option to change the tree view location and make it vertical as it was before.

![Web capture_23-8-2024_0127_127 0 0 1](https://github.com/user-attachments/assets/2f875022-371b-48fc-90c3-5ba88fe020fa)

**Comment by ostap667inbox** [comment, marked as answer]
_Settings > Extra Networks > Extra Networks directory view style (requires Reload UI)_
Set this trigger to 'tree'

p.s. Also install the Lobe Theme plugin in Extensions, which will better utilize the area of your screen. In particular, its LoRA panel is on the side, it can be fixed or slide out from the side.

**Reply by BellZwei** [reply]
Thank you
